src/templates/aws/addons/ssm.test.ts

```diff
@@ -0,0 +1,56 @@
+import { AwsOptions } from '..';
+import { remove } from '../../../helpers/file';
+import applyCommon from './common';
+import applySsm, { ssmModuleContent, ssmVariablesContent } from './ssm';
+
+describe('SSM add-on', () => {
+  describe('given valid AWS options', () => {
+    const projectDir = 'ssm-addon-test';
+
+    beforeAll(() => {
+      const awsOptions: AwsOptions = {
+        projectName: projectDir,
+        provider: 'aws',
+        infrastructureType: 'advanced',
+        awsRegion: 'ap-southeast-1',
+      };
+
+      applyCommon(awsOptions);
+      applySsm(awsOptions);
+    });
+
+    afterAll(() => {
+      jest.clearAllMocks();
+      remove('/', projectDir);
+    });
+
+    it('creates expected files', () => {
+      const expectedFiles = [
+        'main.tf',
+        'providers.tf',
+        'outputs.tf',
+        'variables.tf',
+        'modules/ssm/main.tf',
+        'modules/ssm/variables.tf',
+        'modules/ssm/outputs.tf',
+      ];
+
+      expect(projectDir).toHaveFiles(expectedFiles);
+    });
+
+    it('creates expected folders', () => {
+      expect(projectDir).toHaveDirectory('modules/ssm/');
+    });
```

Comment:
Is the test needed? 
As we are checking at the files `modules/ssm/*`, it seems to me we do not really care if the folder exists or not (in a sense that in case we change the feature and there is no more files in this folder, we do not want to "ensure the empty folder exists") 🤔

Reply:
I just want to make it inside a dedicated test case. But as you said, it's unnecessary. Remove them in https://github.com/nimblehq/infrastructure-templates/pull/77/commits/4758e95537f6e150e7000af18525f7ab72fb3b9c